auth0/resource_auth0_tenant.go

```diff
@@ -285,7 +285,7 @@ func readTenant(ctx context.Context, d *schema.ResourceData, m interface{}) diag
 }
 
 func updateTenant(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
-	tenant := buildTenant(d)
+	tenant := expandTenant(d)
```

Comment:
Renaming so we use the same terminology as the official terraform docs e.g. expanding when constructing the payload to send to the API and flattening when we deconstruct it to store into state.